docs/static/settings/monitoring-settings-legacy.asciidoc

```diff
@@ -28,12 +28,18 @@ Logstash metrics must be routed through your production cluster. You can specify
 a single host as a string, or specify multiple hosts as an array. Defaults to
 `http://localhost:9200`.
 
-NOTE: If your Elasticsearch cluster is configured with dedicated master-eliglble
+NOTE: If your Elasticsearch cluster is configured with dedicated master-eligible
 nodes, Logstash metrics should _not_ be routed to these nodes, as doing so can
 create resource contention and impact the stability of the Elasticsearch
 cluster. Therefore, do not include such nodes in
 `xpack.monitoring.elasticsearch.hosts`.
 
+`xpack.monitoring.elasticsearch.proxy`::
+
+The monitoring {es} instance and monitored Logstash can be separated by proxy,
+to made able Logstash to connect a proxed {es} put the URI of the intermediate proxy
+in this setting. Empty string is treated as if proxy was not set.
+
```

Comment:
What do you think about: 

>"The monitoring {es} instance and monitored Logstash can be separated by a proxy.
>To enable Logstash to connect a proxed {es}, set this value to the URI of the intermediate proxy. 
>An empty string is treated as if proxy was not set."


Will people know where to find the proxy's URI and know the format it should be in?

Reply:
The format is the standard URI format, `<proto>://<path>`. The value should be part of their network configuration, it's nothing we can provide, we consume that.

Reply:
Thank's @karenzone for rewording, much better!